configs/raspberrypi_config

```diff
@@ -91,6 +91,12 @@ function platform_modify() {
     run_and_log_cmd "sudo ln -fs /uboot/overlays work/rootfs/boot/overlays"
     run_and_log_cmd "sudo ln -fs /uboot/$CMDLINE work/rootfs/boot/$CMDLINE"
 
+    # Raspberry Pi headless configuration expects boot partition to be mounted
+    # at /boot, so replace these services with /uboot
+    run_and_log_cmd "find work/rootfs/lib/ -type f -name *.service | xargs sed -i.bak 's|/boot/|/uboot/|g'"
+    log_info "Diff from modified service files:\n$(find work/rootfs/lib/ -type f -name *.service | xargs -I@ bash -c 'diff @.bak @')"
+    run_and_log_cmd "find work/rootfs/lib/ -type f -name *.service.bak | xargs rm"
```

Comment:
This is a bit noisy. I am unsure about the value. The output looks like:

```
2021-03-05 11:41:10 [INFO] [mender-convert-modify] Running hook: platform_modify
2021-03-05 11:41:10 [INFO] [mender-convert-modify] Diff from modified service files:
3c3
< ConditionPathExists=/boot/os_config.json
---
> ConditionPathExists=/uboot/os_config.json
2,3c2,3
< Description=Turn on SSH if /boot/ssh is present
< ConditionPathExistsGlob=/boot/ssh{,.txt}
---
> Description=Turn on SSH if /uboot/ssh is present
> ConditionPathExistsGlob=/uboot/ssh{,.txt}
8c8
< ExecStart=/bin/sh -c "systemctl enable --now ssh && rm -f /boot/ssh ; rm -f /boot/ssh.txt"
---
> ExecStart=/bin/sh -c "systemctl enable --now ssh && rm -f /uboot/ssh ; rm -f /uboot/ssh.txt"
3c3
< ConditionPathExists=/boot/wpa_supplicant.conf
---
> ConditionPathExists=/uboot/wpa_supplicant.conf
10c10
< ExecStart=/bin/mv /boot/wpa_supplicant.conf /etc/wpa_supplicant/wpa_supplicant.conf
---
> ExecStart=/bin/mv /uboot/wpa_supplicant.conf /etc/wpa_supplicant/wpa_supplicant.conf
2021-03-05 11:41:10 [INFO] [mender-convert-modify] Running hook: grow_data_partition
...
```

Reply:
What if you do `run_and_log_cmd` for the diff instead? And then add a `log_info`, with something like:
`Certain service files have been changed to align with our /uboot boot partition mount point. See convert.log for more information.`?

Reply:
+1 to Ole's suggestion!

Reply:
I tried this... the problem is that `convert.log` has disappeared (at least from my runs). But you guys are pointing me to the right direction so I am going to 1) fix whatever issue is there with the file and 2) move the diff to the file only.

Reply:
The log file was there... I just didn't know where :weary: